src/admin/apiAdminCommunication.ts

```diff
@@ -0,0 +1,50 @@
+import 'whatwg-fetch';
```

Comment:
better to move apiAdminCommunication.ts to Services/Api.ts
like 
admin/Components
admin/Services
admin/Actions
...